drivers/lora/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library_named(loramac-node)
```

Comment:
I think this declaration belongs to `loramac-node` repo. @SebastianBoe?

Reply:
This file is built first. We have to declare it here.